components/core/core-database/src/main/java/org/eclipse/dirigible/components/database/DataSourceSystemConfig.java

```diff
@@ -44,13 +45,13 @@ public class DataSourceSystemConfig {
     public HikariDataSource getDataSource() {
         DataSourceProperties dataSourceProperties = new DataSourceProperties();
         dataSourceProperties.setName("SystemDB");
-        dataSourceProperties.setDriverClassName("org.h2.Driver");
-        dataSourceProperties.setUrl("jdbc:h2:file:./target/dirigible/h2/SystemDB");
-        dataSourceProperties.setUsername("sa");
-        dataSourceProperties.setPassword("");
+        dataSourceProperties.setDriverClassName(Configuration.get("DIRIGIBLE_DATASOURCE_SYSTEM_DRIVER", "org.h2.Driver"));
```

Comment:
Let's use prefix DIRIGIBLE_DATABASE_*
and add documentation here:
https://www.dirigible.io/help/setup/setup-environment-variables/
and add definition here:
https://github.com/eclipse/dirigible/blob/abadf3a1259f5c4b4e08983d0913a735bb45644d/modules/commons/commons-config/src/main/java/org/eclipse/dirigible/commons/config/Configuration.java#L504